dashboard/lib/service/appengine_cocoon.dart

```diff
@@ -146,23 +146,33 @@ class AppEngineCocoonService implements CocoonService {
   }
 
   @override
-  Future<bool> rerunTask(Task task, String idToken) async {
-    assert(idToken != null);
+  Future<CocoonResponse<bool>> rerunTask(Task task, String idToken) async {
+    if (idToken == null || idToken.isEmpty) {
+      return const CocoonResponse<bool>.error('Need to sign in to perform rerun operation');
```

Comment:
```suggestion
      return const CocoonResponse<bool>.error('Sign in to trigger reruns');
```

Error messages should be actionable. https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#error-messages-should-be-useful

Reply:
Thanks for the instructions on error messages! I will use more verbs, make them shorter, and explain more accurately.